piker/clearing/_paper_engine.py

```diff
@@ -271,6 +272,25 @@ async def fake_fill(
             dt=pendulum.from_timestamp(fill_time_s),
             bsuid=key,
         )
+
+        # Update in memory ledger per trade
+        ledger_entry = {}
+        ledger_entry[oid] = t.to_dict() 
+        
+        # Store txn in state for PP update
+        self._txn_dict[oid] = t
+        self._trade_ledger.update(ledger_entry)
+        
+        # Write to ledger toml  
+        with open_trade_ledger(self.broker, 'paper') as ledger:
+            ledger.update(self._trade_ledger)   
+
+        # Write to pps toml
+        with open_pps(self.broker, 'paper-id') as table: 
```

Comment:
i would say consider making the *key* just `'paper'` or `piker_paper` (just to be explicit that it's *piker's* engine)?

Reply:
Done!